hr_public_holidays/wizards/public_holidays_next_year_wizard.xml

```diff
@@ -0,0 +1,70 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright 2016 Trobz
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
+
+<odoo>
+
+<record id="public_holidays_next_year_wizard_view" model="ir.ui.view">
+    <field name="name">Create Next Year Public Holidays</field>
+    <field name="model">public.holidays.next.year.wizard</field>
+    <field name="arch" type="xml">
+        <form string="Create Next Year Public Holidays">
+            <sheet>
+                <div>
+                        Use this wizard to create pulic holidays based on the 
+                        existing ones.<br/>
+                        Only the last templates of each country 
+                        will be taken into account (If you select templates 
+                        from 2012 and 2015 of the same country; '
+                        only the templates from 2015 will be taken into 
+                        account).
+                </div>
+                <group>
+                    
+                    <group string="Defaults">
+                        <div>     
+                        Normally, you should not need to input anything in 
+                        optional fields and only need to click on the button 
+                        "Create".<br/>
+                        By default, the most recent public holidays 
+                        for each country are used as template to create  
+                        public holidays for the year following the templates.
+                        </div> 
+                    </group>
+                    <group string="Optional">
+                        <div colspan="2">
+                        The below optional fields are here only to handle 
+                        special situtaions like "2016 was a special year with 
```

Comment:
* s/situtaions/situations
* (The 150th anniversary of Italian unification was actually in 2011, I used 2016 in my example just to make it more current 😁  )

* I would put this group under the first one instead of next to it, it looks clumsy:
![screenshot from 2016-11-07 10-04-54](https://cloud.githubusercontent.com/assets/5588738/20052172/1703f058-a4d3-11e6-920f-6f33b22238c7.png)
Or we could even maybe put the two groups in two notebook pages so that the Optional section is "hidden" unless one explicitly clicks on it; while the "Defaults" explanation is immediately displayed. This could make the wizard look simpler and not overload the user with options/fields, what do you think?